infrastructure/ecr.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_ecr_repository" "main" {
```

Comment:
One of the requirements was to deploy the app in a container. I've set up the infrastructure via Terraform.
Architecture:
* AWS ECS is used to run the app in a Docker container
* A single ECS cluster that contains a single ECS Service that runs a single ECS Task that runs the app
* AWS RDS PostgreSQL is used as a RDMS
* AWS Load Balancer is used to route traffic to the ECS Task
* AWS ECR is used as a Docker containers registry